### DISCUSSION TITLE
Login Screen Bypass Auth0

### DISCUSSION THREAD

**Discussion by Wanhack** [requarks/wiki#6454] [Help / Questions] [answered]
Hello,

Been trying to figure out how to skip the authentication provider step, the only provider is Auth0 on that list, I have hidden the local login screen. For Auth0 I have normal login and Windows/Google login enabled. On a #4099 there is a mention of reordering of the authentication strategies, which I have no clue on how to do. Any help would be appreciated. Just a quick edit: I have the Bypass login Screen option enabled

**Comment by NGPixel** [comment, marked as answer]
Under Admin > Authentication, you need to re-order the strategies so that Auth0 is first (use the handles to the left of each strategy to drag-n-drop)
